src/ReactWrapper.js

```diff
@@ -471,15 +471,15 @@ export default class ReactWrapper {
    * @param {Array} args
    * @returns {ReactWrapper}
    */
-  simulate(event, ...args) {
+  simulate(event, mock) {
```

Comment:
this is its only use https://github.com/facebook/react/blob/master/src/test/ReactTestUtils.js#L583 (in react v15) so I guess `undefined`, `null` or `{}` are all harmless, which one do you like best?